runtime/gc_realtime/MetronomeAlarmThread.cpp

```diff
@@ -163,13 +136,12 @@ MM_MetronomeAlarmThread::metronomeAlarmThreadWrapper(void* userData)
 bool
 MM_MetronomeAlarmThread::startThread(MM_EnvironmentBase *env)
 {
-	J9JavaVM *vm = (J9JavaVM *)env->getLanguageVM();
-	UDATA startPriority;
+	uintptr_t startPriority;
 	bool retCode;
 
 	startPriority = J9THREAD_PRIORITY_MAX;
 
-	if (0 != vm->internalVMFunctions->createThreadWithCategory(
+	if (0 != createThreadWithCategory(
```

Comment:
Would be better to use J9THREAD_SUCCESS instead of zero. This is definition from OMR